src/configcatclient.cpp

```diff
@@ -169,8 +170,9 @@ std::shared_ptr<Value> ConfigCatClient::getValue(const std::string& key, const C
         for (auto keyValue : *settings) {
             keys.emplace_back(keyValue.first);
         }
-        LOG_ERROR << "Value not found for key " << key << ". Here are the available keys: " << keys
-                  << " Returning nullptr.";
+        LOG_ERROR(1001) <<
+            "Failed to evaluate setting '" << key << "' (the key was not found in config JSON). "
+            "Returning nullptr. Available keys: [" << keys << "].";
```

Comment:
[] is not needed here because LogEntry decorates the string vectors with []. 
```suggestion
            "Returning nullptr. Available keys: " << keys << ".";
```